test/integration/sagemaker/test_experiments.py

```diff
@@ -22,7 +22,12 @@
 
 
 @pytest.mark.skip_py2_containers
-def test_training(sagemaker_session, ecr_image, instance_type):
+@pytest.mark.skip_by_py_version
```

Comment:
Why do we require two markers here?

Reply:
skip_py2_containers has no effect, ill remove it

Reply:
It looks like skip_py2_containers, works, since the py2 containers aren't being run. IIRC, the issue was that even in py3 containers, smexperiments wasn't recognized. This might be due to not instaling smexperiments as part of the test dependency.